res/css/_common.scss

```diff
@@ -806,3 +806,54 @@ legend {
     mask-repeat: no-repeat;
     mask-size: contain;
 }
+
+@define-mixin RightPanelCard {
```

Comment:
Because it would change everything on the Basecard like `UserInfo`, etc.

I think the name should be changed to something else to avoid confusion.

Reply:
(it should also be made to work in `mx_BaseCard` given it's name, rather than not using it)

Reply:
I think it is quite reasonable to believe that all of the buttons should be `24px` though I will check internally; is there anything else?

Reply:
I can confirm 24px should be fine for all cards

Reply:
Thanks!

If design input / review is not required, I would incorporate the mixin to `_BaseCard.scss`.

Reply:
> If design input / review is not required, I would incorporate the mixin to `_BaseCard.scss`.

Design has confirmed `24px` is fine internally; is there anything else I should check?

Reply:
Thanks. I think there is not really other stuff to be checked.

I am returning these declarations to the original position in `mx_ThreadPanel` rather than setting them by default.

````
height: 100px;
overflow: visible;
````